concepts/eventing-code-guidelines.md

```diff
@@ -35,7 +35,7 @@ Use [t.Testing](https://pkg.go.dev/testing) for unit tests and for controller in
 
 **Assertion libraries**:
 
-Use [stretchr/testify](https://github.com/stretchr/testify) as assertion library.
+Use [stretchr/testify](https://github.com/stretchr/testify) as assertion library. The controller integration tests are an exception to this and use [onsi/gomega](https://github.com/onsi/gomega) as an assertion library instead.
```

Comment:
(nit)
```suggestion
Use [stretchr/testify](https://github.com/stretchr/testify) as assertion library. The controller integration tests are an exception to this, use [onsi/gomega](https://github.com/onsi/gomega) as an assertion library for them instead.
```

for me it sounds better

Reply:
I am ok with both ways. @NHingerl what do you prefer ?

Reply:
Either the version with the "and", or use a semicolon or full stop instead of the comma to separate the sentences, as they have different subjects.

Reply:
@VladislavPaskar I agree with @NHingerl. I think the subject is not important in this case (whether we or the tests use the assertion library because the outcome is the same).
Can you create an updated suggestion ?

Reply:
@VladislavPaskar As discussed: I will pick your suggestion since the sentence is more clear this way to you.
However I prefer a full stop instead of a comma.

```suggestion
Use [stretchr/testify](https://github.com/stretchr/testify) as assertion library. The controller integration tests are an exception to this. Use [onsi/gomega](https://github.com/onsi/gomega) as an assertion library instead.
```